std/portable/index.js

```diff
@@ -197,15 +204,16 @@ globalScope["isFloat"] = function isFloat(arg) {
   return typeof arg === "number";
 };
 
-globalScope["isReference"] = function isClass(arg) {
-  return typeof arg === "object" || typeof arg === "string";
+globalScope["isReference"] = function isReference(arg) {
+  return typeof arg === "object" || typeof arg === "string" || typeof arg === "symbol";
```

Comment:
Not sure about symbols here. Not really a pointer, and can't really `load` from it.

Reply:
got it. I'll remove it